tensorflow_addons/image/__init__.py

```diff
@@ -28,3 +28,4 @@
 from tensorflow_addons.image.transform_ops import rotate
 from tensorflow_addons.image.transform_ops import transform
 from tensorflow_addons.image.translate_ops import translate
+from tensorflow_addons.image.augment_ops import blend
```

Comment:
@gabrieldemarmiesse Is blend an `augment_ops` or something else? I.e. in Opencv is a `addWeighted` operation so I don't know if could be in the trasnform group, an `arithmetic` etc. 
I we will add more blending operations we could use `image.blend_ops`.

Reply:
Or we could use `image.compose_ios` semantically like [ImageMagick](http://www.imagemagick.org/Usage/compose/)

Reply:
This is not critical, we can always refactor/rename files later on as this does not appear in the public API. Let's keep it like that now, we can change it later on.

Reply:
Ok but it is better to give them a more appropriate default name. `compose` or `compositing` for blend is better then `augment_ops`.